cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/helpers/SubgraphWalker.kt

```diff
@@ -455,20 +455,23 @@ private fun CallExpression.duplicateTo(call: CallExpression, callee: Reference)
 fun MemberCallExpression.toCallExpression(callee: Reference): CallExpression {
     val call = CallExpression()
     duplicateTo(call, callee)
+    call.arguments.forEach { it.astParent = call }
 
     return call
 }
 
 fun CallExpression.toMemberCallExpression(callee: MemberExpression): MemberCallExpression {
     val call = MemberCallExpression()
     duplicateTo(call, callee)
+    call.arguments.forEach { it.astParent = call }
```

Comment:
Isnt't this the same thing we are doing at the callee of this function? If we look at the internals of dublicateTo, they appear to copy the argumentEdges. Would it be possible to set the ast Parent in there?

Reply:
it was also my impression that we (should) already do this inside the duplicate function. Let me have a Quick Look before we merge this please.

Reply:
It sets the ast parent only in some cases